crates/rspack_core/src/incremental/mutations.rs

```diff
@@ -17,13 +18,18 @@ pub struct Mutations {
   affected_modules_with_module_graph: OnceCell<IdentifierSet>,
   affected_modules_with_chunk_graph: OnceCell<IdentifierSet>,
   modules_with_chunk_graph_cache: IdentifierDashMap<Option<u64>>,
+  affected_chunks_with_chunk_graph: OnceCell<UkeySet<ChunkUkey>>,
 }
 
 #[derive(Debug)]
 pub enum Mutation {
   ModuleBuild { module: ModuleIdentifier },
   ModuleRevoke { module: ModuleIdentifier },
   ModuleSetAsync { module: ModuleIdentifier },
+  ChunkAdd { chunk: ChunkUkey },
+  ChunkSplit { from: ChunkUkey, to: ChunkUkey },
```

Comment:
will this created by SplitChunksPlugin ?

Reply:
Yes